index.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 var validations = {
-	'authentication' : require( './authentication' )
+	'authentication' : require( './authentication' ),
+	'legacy-service' : require( './legacy-service' )
```

Comment:
I'd probably just do `users` since this could be used in a bunch of services that require `userId`. e.g. `validation.users.complete( `Id` )`;